ci_framework/roles/devscripts/tasks/04_overlays.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Get OCP related nodes
+  register: _current_vms
+  community.libvirt.virt:
+    command: "list_vms"
+    uri: "qemu:///system"
+
+- name: Stop OCP Masters
```

Comment:
nit: Since we are also looking at workers ... can we state `nodes` instead of `masters`